group_vars/bms.yml

```diff
@@ -0,0 +1,5 @@
+oneview_config_hostname: synergy.cloudra.local
```

Comment:
I have moved these settngs to group_vars/all/vars because one of the playbooks needs them and does not target BM machines but the ansible controller. (ie localhost). Which means we don;t need the oneview_ variables defined here any more

Reply:
Fine, I moved mine to group_vars/all/vars and updated the sample file.